ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```diff
@@ -186,3 +186,90 @@
 .govuk-form-group {
   margin-bottom: 0;
 }
+
+.govuk-header {
+  border-bottom: none;
+}
+
+.govuk-header__link--homepage:hover,
+.govuk-header__link--homepage:active {
+  border-bottom: $space-1 solid $color-black;
+}
+
+.govuk-header__navigation-item--active a,
+.govuk-header__navigation-item--active a:hover,
+.govuk-header__navigation-item--active a:visited {
+  color: $color-yellow;
+  font-weight: bold !important;
+
+  &:focus {
+    background-color: transparent;
+    box-shadow: none;
+  }
+}
+
+.govuk-header__navigation-item--active .govuk-header__link,
+.govuk-header__navigation-item .govuk-header__link:hover {
+  color: $color-yellow;
+  font-weight: bold;
+  border-bottom: $space-1 solid $color-yellow;
+  text-decoration: none;
+}
+
+.govuk-header__navigation-item .govuk-header__link {
+  font-size: 0.875rem;
+  line-height: $typography-md-line-height;
+  font-weight: normal;
+  white-space: nowrap;
+  text-decoration: none;
+  display: inline-block;
+
+  &:after {
+    display: block;
+    content: attr(title);
+    font-weight: bold;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+}
+
+a.govuk-header__link {
+  border-bottom: 1px solid $color-white;
```

Comment:
Due to the bold on hover affect we have, the border-bottom looks a bit weird:
<img width="151" alt="image" src="https://github.com/nationalarchives/ds-caselaw-public-ui/assets/173052089/95f8b793-f2e1-4a42-8b03-1bd1f94a0691">

There's 2 ways I can think of to resolve this:

 - Use `text-decoration: underline` - do you know why we decided to change to border bottom instead of text-decoration? for when we don't have the big yellow border bottom it does look fine with `text-decoration: underline`. If the issue is we want to move the underline down slightly, we can use `text-underline-offset` to move it down a bit.
 - Centerally aligned the text. This does mean we would still have the extra lines on the left and right though:

<img width="158" alt="image" src="https://github.com/nationalarchives/ds-caselaw-public-ui/assets/173052089/9208c1c2-068e-4ce7-bad1-bb59f98665f9">